gensrc/proto/tablet_schema.proto

```diff
@@ -66,6 +67,7 @@ message TabletSchemaPB {
     optional uint32 next_column_unique_id = 7; // OLAPHeaderMessage.next_column_unique_id
     optional bool DEPRECATED_is_in_memory = 8 [default=false];
     optional int64 deprecated_id = 9; // deprecated
+    optional CompressionTypePB compression_type = 10 [default=LZ4_FRAME];
```

Comment:
what's the difference between compression_type and compress_kind

Reply:
compress_kind is a deprecated variable for the storgae engine.